dataland-frontend/src/App.vue

```diff
@@ -65,12 +65,3 @@ export default defineComponent({
   },
 });
 </script>
-
```

Comment:
I put the comment here - even though it has nothing to do with this part of the code.
![image](https://user-images.githubusercontent.com/47553592/210356465-6642ff75-d0b3-44e6-898a-661f9823861b.png)
The colors differ of this box on the left side

Reply:
understood - it's as in the figma